src/menu.cc

```diff
@@ -7,7 +7,7 @@
 Menu::Menu() {
   auto accels=Config::get().menu.keys;
   for(auto &accel: accels) {
-#ifdef JUCI_UBUNTU_BUGGED_MENU
+#ifdef JUCI_UBUNTU
     size_t pos=0;
```

Comment:
The example is not taking into account that the keybindings can be changed. Setting the keybindings in the XML or using source code should not matter. On Ubuntu, just checked, changing keybindings do not take effect until restart. However, changing keybindings on non-Ubuntu systems works perfectly while running juCi++.